.travis.yml

```diff
@@ -2,6 +2,8 @@ language: android
 
 android:
   components:
+    - platform-tools
+    - tools
```

Comment:
You should actually flip these. `tools` is needed first to install the latest `platform-tools` (which I'll update separately).